lib/helpscout/access_token.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Helpscout
+  class AccessToken
+    ROUTE = 'v2/oauth2/token'
+
+    class << self
+      def create # rubocop:disable Metrics/MethodLength
+        response = Helpscout.api.post(
+          create_route,
+          grant_type: 'client_credentials',
+          client_id: Helpscout.app_id,
+          client_secret: Helpscout.app_secret
+        ).body
+
+        Helpscout::AccessToken.new(
+          response[:access_token],
+          response[:expires_in]
+        ).tap do |access_token|
+          Helpscout.configure { |c| c.access_token = access_token }
```

Comment:
We could set the access_token directly on the configuration, maybe adding a `Helpscout.access_token=` method would be beneficial.
```suggestion
          Helpscout.configuration.access_token = access_token
```